protocols/v2/codec-sv2/src/decoder.rs

```diff
@@ -157,17 +233,24 @@ impl<'a, T: Serialize + GetSize + Deserialize<'a>, B: IsBuffer + AeadBuffer> Wit
         frame.into()
     }
 
+    /// Provides a writable buffer for incoming data.
     #[inline]
     pub fn writable(&mut self) -> &mut [u8] {
         self.noise_buffer.get_writable(self.missing_noise_b)
     }
+
+    /// Checks if the buffers are droppable.
```

Comment:
maybe we can link the Buffer trait docs (now is undocumented but as soon as we will document it we will have a meaningful link

Reply:
https://docs.rs/buffer_sv2/1.1.0/buffer_sv2/